markdown.go

```diff
@@ -110,18 +109,31 @@ flag is the %s query parameter below:
 }
 `+"```"+`
 
-### HTTP Status Codes
+::: tip
+Some arguments may belong only to the CLI but appear here too. These usually
+belong to client-side processing of input, particularly in the `+"`add`"+` command.
+:::
+
+
+## HTTP status codes
 
 Status codes used at the RPC layer are simple:
```

Comment:
may want to note that, on error, we either return 500 or 200 + a trailing error.